Handle missing or stale IERS-A better (#13052)
astropy/astropy: docs/whatsnew/5.1.rst

  via the `scipy.optimize.least_squares` function.
* :class:`~astropy.modeling.fitting.DogBoxLSQFitter`, which uses the dogleg algorithm.

.. _whatsnew-iers-handling:

Allow time conversions without predictive Earth rotation data (IERS-A)
======================================================================

Some time conversions like UTC -> UT1 require additional Earth rotation data for
full accuracy. These data are provided by the online IERS service as the IERS-A
tables and are downloaded as required. In some use cases this download is not
desired or possible. Examples include an application where full accuracy is not

> required, running on a cluster, or the rare instances when the IERS server and

Reviewer: ```suggestion
required, running on a cluster node without internet access, or the rare instances when the IERS server and
```

mirror are not available. For these cases there is a new config item

> `iers.conf.iers_degraded_accuracy <astropy.utils.iers.Conf.iers_degraded_accuracy>` which specifies the behavior when times are

Reviewer 2: Nitpick: formatting would be nicer with consistent line length (also for changelog item)

Author: Not my fault exactly, that came from accepting review commit suggestions.

Author: I'll fix since there will be at least another commit.

> `iers.conf.iers_degraded_accuracy <astropy.utils.iers.Conf.iers_degraded_accuracy>` which specifies the behavior when times are

Reviewer: ```suggestion
`iers.conf.iers_degraded_accuracy <astropy.utils.iers.Conf.iers_degraded_accuracy>`__ that specifies the behavior when times are
```

Author: Done.

> ``iers.conf.iers_degraded_accuracy`` which specifies the behavior when times are
> outside the range of available IERS data. The options are ``'error'`` (default
> to raise an exception), ``'warn'`` (issue a ``IERSDegradedAccuracyWarning``) or

Reviewer 3: ```suggestion
`iers.conf.iers_degraded_accuracy <astropy.utils.iers.Conf.iers_degraded_accuracy>` which specifies the behavior when times are
outside the range of available IERS data. The options are ``'error'`` (default
to raise an :class:`~astropy.utils.iers.IERSRangeError`), ``'warn'`` (issue an class:`~astropy.utils.iers.IERSDegradedAccuracyWarning`) or
```
